stablehlo/tests/interpret_dot_general.mlir

```diff
@@ -0,0 +1,103 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
```

Comment:
Also, we need tests for all different control flow paths through the implementation. Just one test is not going to be sufficient.